src/components/LinterMessage/styles.module.scss

```diff
@@ -1,3 +1,43 @@
+$top-arrow-width: 10px;
+$top-arrow-height: 20px;
+
+.inline:first-child {
+  &:after,
+  &:before {
```

Comment:
Just out of curiosity, what makes this render as a half triangle? Typically, a `transform` would be applied to a square to rotate it but I don't see that happening.

Reply:
Here is a nice explanation: https://codepen.io/chriscoyier/pen/lotjh

Reply:
Cool.